jenkins/scripts/select-compiler.sh

```diff
@@ -20,7 +20,12 @@ if [ "$SELECT_ARCH" = "PPC64LE" ]; then
 
   echo "Setting compiler for Node version $NODEJS_MAJOR_VERSION on ppc64le"
 
-  if [ "$NODEJS_MAJOR_VERSION" -gt "9" ]; then
+  if [ "$NODEJS_MAJOR_VERSION" -gt "11" ]; then
+    # XXX(@sam-github) Why? I didn't need binutils in path to build/link/test
+    # node, the system default of 2.24 worked fine for me.
+    export PATH=/usr/lib/binutils-2.26/bin/:$PATH
```

Comment:
Maybe it's required only for a subset of platforms (also IIUC GCC > 5 is better at backwards compatibility)? Let's try without it.

Reply:
I'd prefer to figure that out as a separate step. We needed to do it separately when we introduced the 4.9.4 compiler and I don't want to break anything. @refack if you want we can make it conditional on PPC if you disagree?

Reply:
Ack.